workspace/__lib__/xod/common-hardware/button/patch.xodp

```diff
@@ -1,70 +1,48 @@
 {
-  "description": "Reads a generic button or another mechanical switch. It is expected that the button is low while pressed, the on-board pull-up resistor is enabled if possible. The node provides signal debounce with 20 ms settle delay.\n\nPossible errors:\n— Invalid port",
+  "description": "Reads a generic button or another mechanical switch. It is expected that the button is normally high, i.e. it is pulled up with a resistor. The node provides signal debounce with 20 ms settle delay.\n\nPossible errors:\n— Invalid port",
```

Comment:
The text before the change is correct. Wrong merge/rebase?

Reply:
Hm... very strange. Revert it.